examples/complete/variables.tf

```
@@ -3,7 +3,12 @@ variable "region" {
   description = "AWS Region"
 }
 
-variable "example" {
-  type        = string
-  description = "The value which will be passed to the example module"
+# TODO: use `optional` when it becomes GA
+variable "service_quotas" {
+  type        = any
```

```suggestion
  type        = list(map(any))
```

or `list(any)`

since we know this is a list

---

A `list(map(any))` may not work since the `map` type enforces the same key structure in each item. You may have to use a `list(any)` instead. For example, we want to support both quote_code and quote_name so some items may exclusively use one of the other which means the key structure will be different per item which may trigger the error.

---

I went with `list(any)`